ui/apps/platform/src/Containers/Clusters/InitBundles/InitBundleWizardStep2.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement } from 'react';
+import { Alert, Flex, Title } from '@patternfly/react-core';
+
+import { InitBundleWizardFormikProps } from './InitBundleWizard.utils';
+
+export type InitBundleWizardStep2Props = {
+    errorMessage: string;
+    formik: InitBundleWizardFormikProps;
+};
+
+function InitBundleWizardStep2({ errorMessage, formik }: InitBundleWizardStep2Props): ReactElement {
+    const { values } = formik;
+
+    /* eslint-disable no-nested-ternary */
```

Comment:
Is there a reason for disabling this as opposed to pulling the logic out into a function?

```javascript
function renderBundleInstructions() {
    if (values.installation === 'Operator') {
        return 'Use this bundle to install secured cluster services on OpenShift with an Operator.';
    } else if (values.platform === 'OpenShift') {
        return 'Use this bundle to install secured cluster services on OpenShift with a Helm chart.';
    } else {
        return 'Use this bundle to install secured cluster services on xKS with a Helm chart.';
    }
}

```

Reply:
Super suggestion!

Reply:
Aha, now I remember: there will be 2 versions of content for Operator and Helm installation methods.
* Operator will have the first sentence.
* Helm will have either the second or third (that is, ordinary ternary).

So my next step (pardon pun) will solve the problem.